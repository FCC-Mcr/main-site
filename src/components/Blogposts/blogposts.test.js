import React from "react"
import { render } from "@testing-library/react"
import "@testing-library/jest-dom/extend-expect"

import BlogPosts from "./"

describe("BlogPosts", () => {
  test("renders empty when passed no data", () => {
    const { container } = render(<BlogPosts />)
    expect(container).toMatchSnapshot()
  })
})
