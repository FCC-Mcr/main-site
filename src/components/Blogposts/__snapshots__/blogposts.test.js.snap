// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BlogPosts renders empty when passed no data 1`] = `
<div>
  <div
    class="blogposts "
  >
    <h1 />
  </div>
</div>
`;
